modules/migration-mtc-release-notes-1-8-4.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-4_{context}"]
+= {mtc-full} 1.8.4 release notes
+
+[id=technical-changes-1-8-4_{context}]
+== Technical changes
+
+{mtc-full} ({mtc-short}) 1.8.4 has the following technical changes:
+
+
+
+[id="resolved-issues-1-8-4_{context}"]
+== Resolved issues
+
+{mtc-full} ({mtc-short}) 1.8.4 has the following major resolved issues:
+
+.UI stuck at Namespaces while creating a migration plan
+
+When trying to create a migration plan from the {mtc-short} UI, the migration plan wizard becomes stuck at the *Namespaces* step. This issue has been resolved in {mtc-short} 1.8.4. link:https://issues.redhat.com/browse/MIG-1597[(MIG-1597)]
+
+.Migration is stuck as Direct Volume Migration fails with an `InvalidPVCs` error
+
+When performing a migration, the migration becomes stuck at the Direct Volume Migration (DVM) step, and before finally failing. link:https://issues.redhat.com/browse/MIG-1595[(MIG-1595)]
+
+.Migration fails with error of no matches for kind Virtual machine in version kubevirt/v1
+
+During the migration of an application, all the necessary steps, including the backup, DVM, and restore, are successfully completed. However, the migration is marked as unsuccessful with the error message `no matches for kind Virtual machine in version kubevirt/v1`. link:https://issues.redhat.com/browse/MIG-1594[(MIG-1594)]
```

Comment:
![image](https://github.com/user-attachments/assets/fefac413-d500-465a-904e-12ad74275e77)

Preview looks incorrect.

Reply:
@stillalearner - yeah the preview is not updating

![image](https://github.com/user-attachments/assets/a3cb6249-177d-4124-bdad-1c131952f525)

I will have a look at this later today

Reply:
Updated the preview
![image](https://github.com/user-attachments/assets/4ca1d6ea-7d28-4554-a295-b497f814d0e4)